tests/test_downloader_handlers.py

```diff
@@ -420,6 +420,26 @@ def _clean(data):
         if errback:
             deferred.addErrback(errback)
         return deferred
+    
+    def test_ftp_list_success(self):
+        request = Request(url="ftp://127.0.0.1:%s/" % self.portNum,
```

Comment:
Added the test on https://github.com/bernardotorres/scrapy/commit/1c862e1f6a442cb69ec0b307f08a336d660e4705